### ISSUE TITLE
How to change activity indicator color?

### ISSUE THREAD

**Issue by yinanfang** [jdg/MBProgressHUD#383]
I saw the discussion on the earlier thread #313. I wasn't intent to use a custom view instead of the activity indicator. I just want to simply change the color. #313 mentioned that hud.activityIndicatorColor is deprecated. What's the new property for it?

**Comment by yas375** [comment]
Interesting. Per [the deprecation warning](https://github.com/jdg/MBProgressHUD/blob/f3732cc1058da7d9e9856a27c5e476f34fb28673/MBProgressHUD.h#L419) I would assume you should use

```objective-c
[[UIActivityIndicatorView appearanceWhenContainedIn:[MBProgressHUD class], nil] setColor:[UIColor orangeColor]];
```

But [this line](https://github.com/jdg/MBProgressHUD/blob/f3732cc1058da7d9e9856a27c5e476f34fb28673/MBProgressHUD.m#L447) overrides it and `MBProgressHUD#contentColor` will be used. But setting `#contentColor` will apply to text and to activity indicator at the same time...

**Comment by matej** [comment]
Yeah, that's not good. I think the best option here might be to expose `indicator` than you can cast it to the correct view type and customize it.

**Comment by yinanfang** [comment]
@yas375 Thanks for providing the solution. I think @matej has a valid point. It would be great if we can have an alternative that's similar to the deprecated `hud.activityIndicatorColor`

**Comment by jjxtra** [comment]
This is badly needed. Rolled back to an old version because it's basically impossible to change the activity indicator background color now.

**Comment by matej** [comment]
Should be fixed in f663584

**Comment by jjxtra** [comment]
Xcellent
